title_belt_nhl/main.py

```diff
@@ -1,23 +1,24 @@
 import csv
-from datetime import date
+from datetime import date, datetime
 from pathlib import Path
 from textwrap import dedent
 from typing import Union
+from title_belt_nhl.service.nhl_api import getFullSchedule
+from title_belt_nhl.models.nhl_team_schedule_response import Game
 
 import click
 
 EXCEL_EPOCH_DATE = date(1900, 1, 1)
-SCHEDULE_FILE = Path(__file__).parent / "static" / "schedule_2024_2025.csv"
-
+SEASON = '20242025'
+INITIAL_BELT_HOLDER = 'FLA'
 
 @click.command()
 @click.option("--team", default="VAN", required=True)
-@click.option("--holder", "--belt-holder", default="PIT", required=True)
```

Comment:
Would it make sense to change this to the initial belt holder so you don't need to use a global var?